content/v2.0/query-data/flux/sql.md

```diff
@@ -154,6 +155,77 @@ Use the variable to manipulate queries in your dashboards.
 
 ---
 
+## Use secrets to populate connection credentials
+If your SQL database requires authentication, use [InfluxDB secrets](/v2.0/security/secrets/)
+to store and populate sensitive connection credentials for your database.
```

Comment:
Maybe: "to store and populate ~sensitive~ connection credentials ~for your database~."